chaincfg/init_test.go

```diff
@@ -179,21 +179,21 @@ var (
 				Id:          "Abstain",
 				Description: "Abstain voting for Pedro",
 				Bits:        0x0, // 0b0000
-				IsIgnore:    true,
+				IsAbstain:   true,
 				IsNo:        false,
 			},
 			{
 				Id:          "Yes",
 				Description: "Vote for Pedro",
 				Bits:        0x2,  // 0b0010
-				IsIgnore:    true, // XXX this is the invalid choice
+				IsAbstain:   true, // XXX this is the invalid choice
```

Comment:
In this case, the `Bits` field is not 0, so having the `IsAbstain` flag set to true is an error.  @marcopeereboom can correct me if I'm wrong, but I believe the intent here is to ensure the choices are consistently defined across agendas.  That is to say that having no bits set for a particular agenda is always the abstaining choice.